internal/common/retrystrategy/retry_state.go

```diff
@@ -1,11 +1,17 @@
 package retrystrategy
 
 const (
-	RetryStrategyPendingState   = "PENDING"
-	RetryStrategyCompletedState = "COMPLETED"
-	RetryStrategyErrorState     = "ERROR"
-	RetryStrategyPausedState    = "PAUSED"
-	RetryStrategyUpdatingState  = "UPDATING"
-	RetryStrategyIdleState      = "IDLE"
-	RetryStrategyDeletedState   = "DELETED"
+	RetryStrategyPendingState    = "PENDING"
+	RetryStrategyCompletedState  = "COMPLETED"
+	RetryStrategyErrorState      = "ERROR"
+	RetryStrategyPausedState     = "PAUSED"
+	RetryStrategyUpdatingState   = "UPDATING"
+	RetryStrategyInitiatingState = "INITIATING"
+	RetryStrategyIdleState       = "IDLE"
+	RetryStrategyFailedState     = "FAILED"
+	RetryStrategyActiveState     = "ACTIVE"
+	RetryStrategyDeletedState    = "DELETED"
```

Comment:
https://github.com/10gen/mms/pull/106846 API team is adding DELETING status FYI added you as a reviewer